fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb

```diff
@@ -94,7 +94,9 @@ def self.details
         [
           "This action allows you to upload symbolication files to Crashlytics.",
           "It's extra useful if you use it to download the latest dSYM files from Apple when you",
-          "use Bitcode"
+          "use Bitcode. This action will not fail the build if one of the uploads failed.",
+          "The reason for that is that sometimes some of dSYM files are invalid, and we don't want",
```

Comment:
How do we know  a dSYM is invalid ? Is there any way to validate them ? 
Do we know how they become invalid ?